fake_service/tests/test_api.py

```diff
@@ -6,47 +6,60 @@
 
 
 class SchemaApiTestCase(APITestCase):
-
     def setUp(self):
         self.user = User.objects.create(username="alabama", password="password")
         refresh = RefreshToken.for_user(self.user)
-        self.client.credentials(HTTP_AUTHORIZATION=f'Bearer {refresh.access_token}')
-        self.post = Schema.objects.create(user=self.user,
-                                          name="qwerty228",
-                                          column_separator=".",
-                                          string_character="'",
-                                          fields=[
-                                              {
-                                                  "name": "qwertty228",
-                                                  "type": "integer",
-                                                  "value_from": "20",
-                                                  "value_to": "34"
-                                              }
-                                          ])
+        self.client.credentials(HTTP_AUTHORIZATION=f"Bearer {refresh.access_token}")
```

Comment:
i think [this](https://www.django-rest-framework.org/api-guide/testing/#authenticating) is better way to do this